_docs/getting-started/csdp-introduction.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
This page should be added to nav.yml as well.

Reply:
Will do; not complete at the moment